### PR TITLE
TextBox Fix Tapping the clear button triggers the valueChanged event (T812448)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js [hunk 1]
```diff
@@ -754,6 +754,31 @@ QUnit.module("options changing", moduleConfig, () => {
         }).trigger(dxPointerDown);
     });
 
+    QUnit.test("tap on clear button should not raise onValueChange event (T812448)", (assert) => {
+        let callCount = 0;
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]: `const valueChangeStub = sinon.stub`
<...>
`onValueChanged: valueChangeStub`
<...>
You can check `valueChangeStub.callCount`(number) or `valueChangeStub.calledOnce`(bool)

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]: Thanks, value change counter is replaced by stub now.

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js [hunk 2]
```diff
@@ -754,6 +754,29 @@ QUnit.module("options changing", moduleConfig, () => {
...
+
+        const dxPointerDown = $.Event("dxpointerdown");
+        dxPointerDown.pointerType = "touch";
+        $clearButton.on("dxpointerdown", e => {
+            assert.equal(this.input.val(), "", "texteditor is empty");
+        }).trigger(dxPointerDown);
+
+        kb.type("change")
+            .change();
+        assert.equal(valueChangeStub.calledOnce, 1, "onValueChanged was called once");
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]: calledOnce: **boolean**
```suggestion
        assert.ok(valueChangeStub.calledOnce, "onValueChanged was called once");
```

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js [hunk 3]
```diff
@@ -754,6 +754,29 @@ QUnit.module("options changing", moduleConfig, () => {
...
+        });
+        const $clearButton = $element.find(CLEAR_BUTTON_SELECTOR).eq(0);
+        const kb = this.keyboard;
+        kb.type("clear");
+        assert.equal(this.input.val(), "clear", "texteditor has correct value");
+
+        const dxPointerDown = $.Event("dxpointerdown");
+        dxPointerDown.pointerType = "touch";
+        $clearButton.on("dxpointerdown", e => {
+            assert.equal(this.input.val(), "", "texteditor is empty");
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]:
```suggestion
            assert.strictEqual(this.input.val(), "", "texteditor is empty");
```

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**